packages/cadl-ranch-specs/http/collectionFormat/main.cadl

```diff
@@ -0,0 +1,57 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Rest;
+using Cadl.Http;
+
+@service({
+  title: "Collection Format Testing",
+  version: "2022-12-16-preview",
+})
+@doc("Test for collectionFormat.")
+@supportedBy("dpg")
+@scenarioService("/collectionFormat")
+namespace CollectionFormat;
+
+alias MultiCollectionFormatParameter = {
+  @doc("Possible values for colors are [blue,red,green]")
+  @query({
+    format: "multi",
+  })
+  colors: string[];
+};
+
+alias CsvCollectionFormatParameter = {
+  @doc("Possible values for colors are [blue,red,green]")
+  @query({
+    format: "csv",
+  })
+  colors: string[];
+};
+
+alias DefaultCollectionFormatParameter = {
+  @doc("Possible values for colors are [blue,red,green]")
+  @query
+  colors: string[];
+};
+
+@scenario
+@scenarioDoc("""
+This test is testing sending a multi collection format array query parameters
+""")
+@route("/multi")
+op testMulti(...MultiCollectionFormatParameter): string;
+
+@scenario
+@scenarioDoc("""
+This test is testing sending a csv collection format array query parameters
+""")
+@route("/csv")
+op testCsv(...CsvCollectionFormatParameter): string;
+
+// @scenario
```

Comment:
why is default commented out? I'll go ahead and merge and release, but we should have this at some point